packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/ReactBaseTextShadowNode.java

```diff
@@ -319,6 +327,8 @@ protected Spannable spannedFromShadowNode(
   protected boolean mIsBackgroundColorSet = false;
   protected int mBackgroundColor;
 
+  protected int[] mGradientColors = new int[0];
```

Comment:
maybe just leave it as null so we don't pay any added memory cost for the majority case where gradient colors are not being used.